specificatie/zakelijk-gerechtigden.yaml

```diff
@@ -80,6 +80,13 @@ paths:
           schema:
             $ref: "domain.yaml#/components/schemas/TypeGerechtigdeEnum"
         - $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/parameters/acceptCrs"
+        - in: query
+          name: inclusiefUitsplitsing
```

Comment:
inclusiefUitsplitsing => inclusiefUitSplitsing?

Reply:
hoeveel niveau's diep willen we gaan als inclusiefUitSplitsing gelijk is aan true?

Reply:
Aangepast

Reply:
> hoeveel niveau's diep willen we gaan als inclusiefUitSplitsing gelijk is aan true?

Lijkt mij 1 niveau diep. Als dat ook weer rechten oplevert die gesplitst zijn staat de indicatieZakelijkGerechtigdenUitSplitsing ook daar weer op True en krijg je weer een linksvnaar het volgende niveau aangeboden.

Reply:
we hebben vorige week besproken dat we alles krijgen, dus niet 1 diep, maar ook uit ondersplitsingen.